crates/ethportal-api/src/assets/test/proofs/README.md

```diff
@@ -0,0 +1,10 @@
+# Proofs
+
+### Test assets for post-merge proof (HeaderWithProof) generation
+- Each subdirectory is named by it's block height and contains assets needed to test proof generation utils
+- pre-capella blocks:
+  - `block.ssz` -> ssz encoded beacon block
+  - `hb.ssz` -> ssz encoded historical batch
```

Comment:
Why not just call it historical_batch.ssz, then people can understand what the file is without reading this